app/src/main/java/com/example/cocktaildb/screen/auth/SignInActivity.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.cocktaildb.screen.auth
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import com.example.cocktaildb.MainActivity
+import com.example.cocktaildb.R
+import com.example.cocktaildb.data.repository.AuthRepository
+
+class SignInActivity : AppCompatActivity(), AuthContract.View {
```

Comment:
```suggestion
class SignInActivity : BaseActivity<ActivityLoginBinding>(), AuthContract.View {
```
kế thừa từ base activity & base fragment